nntrainer/layers/dropout.cpp

```diff
@@ -26,9 +26,9 @@ void DropOutLayer::finalize(InitLayerContext &context) {
 
   mask_idx.reserve(input_dims.size());
   for (auto &t : input_dims) {
-    mask_idx.push_back(context.requestTensor(
-      t, context.getName() + ":Mask", Tensor::Initializer::NONE, false,
-      TensorLifespan::ITERATION_LIFESPAN));
+    mask_idx.push_back(
+      context.requestTensor(t, "Mask", Tensor::Initializer::NONE, false,
```

Comment:
not related to the PR it self but maybe change "Mask -> "mask"??

Reply:
I will update here itself.